jbcli/jbcli/cli/jb.py

```diff
@@ -364,6 +365,97 @@ def populate_env_with_secrets():
     return env
 
 
+@cli.command()
+@click.option('--noupdate', default=False,
+              help='Do not automatically update Docker image on start',
+              is_flag=True)
+@click.option('--noupgrade', default=False,
+              help='Do not automatically update jb and generators on start',
+              is_flag=True)
+@click.pass_context
+def freshstart(ctx, noupdate, noupgrade):
+    """Configure the environment and start Juicebox
+    """
+    path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..'))
+    os.chdir(path)
+    click.echo("Welcome to devlandia!")
+    click.echo('Gathering data on available environments...\n')
+    stuff = dockerutil.image_list(print_flag=False, semantic=False)
+
+    interesting_tags = ['develop', 'stable']
+
+    test_tag = stash.get('jb_select')
+    if test_tag and test_tag not in interesting_tags:
+        interesting_tags.append(test_tag)
+        
+    # Sort in ascending order of timestamp
+    extra_lookup = {}
+    stuff.sort(key=lambda x: x[2])
+    for prev_row, row in zip(stuff, stuff[1:]):
+        tag = row[0]
+        prevtag = prev_row[0]
+        readable_timediff = human_readable_timediff(row[2])
+        if tag in interesting_tags:
+            if tag == 'stable':
+                extra_lookup['stable'] = 'stable - ({}) published {}'.format(prevtag, readable_timediff)
+            if tag == test_tag:
+                extra_lookup['test'] = 'test - ({}) published {} (change this with jb select)'.format(test_tag, readable_timediff)
+            if tag == 'develop':
+                extra_lookup['dev'] = 'dev - (develop) published {}'.format(readable_timediff)
+                extra_lookup['core'] = 'core - (develop) published {}'.format(readable_timediff)
+    if 'test' not in extra_lookup:
+        extra_lookup['test'] = 'test - set with `jb select`'
+
+    click.echo()
+
+    env_choices = [{'name': extra_lookup.get('stable', 'stable'),'value': 'stable'},
+                   {'name': extra_lookup.get('dev', 'dev'), 'value': 'dev'},
+                   {'name': extra_lookup.get('core', 'core'), 'value': 'core'},
+                   {'name': extra_lookup.get('test', 'test'), 'value': 'test'},
+                   'hstm-stable', 'hstm-dev', 'hstm-core', 'hstm-test']
+    questions = [
+        {
+            'type': 'list',
+            'name': 'Environment',
+            'message': 'what environment would you like to start?',
+            'choices': env_choices
+        }
+    ]
+
+    response = prompt(questions)
+    env = response['Environment']
+    click.echo(response)
+
+    if not noupgrade:
+        ctx.invoke(upgrade)
+        os.chdir("./environments/{}".format(env))
+    else:
+        os.chdir("./environments/{}".format(env))
+
```

Comment:
I think we just call the start command here.

So we reduce the complexity of what we're doing here to just picking an environment.

```
start(ctx, noupdate, noupgrade)
```

Reply:
The `if not noupgrade` check here is redundant since we're using ctx.forward below, that check will run when start runs, so right now this results in trying to install requirements twice.